### PR TITLE
Travis setup and model tests

### LINKED ISSUE #6: Travis CI for BE
goals for a 4: App has CI and successfully builds by day 5 and is deployed successfully after every PR merges. Logs should show that failed builds are very quickly addressed.

### LINKED ISSUE #19: Test for /jobs/<id> endpoint
---
Name: Test for /jobs/<id> endpoint
About: Test for /jobs/<id>  endpoint to ensure it returns all fields the FE needs to consume and display. https://docs.google.com/spreadsheets/d/1mxnKK4iQ-wrJeZTF3zWcYskbkqlm4G4S5A7gXrfFMy8/edit#gid=483454443

---

### PR DESCRIPTION
**Pull request by Strompy**
# Description
Travis CI/CD up and running
Test file using Pytest and a coverage report.
Tests for Jobs being created and testing that they h ave the correct attributes

# Closes issue(s)
Closes #6 
 
# How to test / reproduce
Run with coverage report:
$ pytest --cov=app tests/  

Without coverage:
$ pytest 

# Screenshots

Screen Shot 2020-10-29 at 3 06 09 PM

# Changes include
- [ ] Bugfix (non-breaking change that solves an issue)
- [X] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Checklist
- [X] I have tested this code
- [X] I have updated the Readme

# Other comments

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -10,3 +10,5 @@ six
 flask-testing
 gunicorn==20.0.4
 python-dotenv
+pytest
+pytest-cov
```
Comment (janegreene) [inline requirements.txt:14]: Allows us to generate a coverage report. Readme already updated!

File: tests/test_jobs.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import json
+import pytest
+from app import db, JobsModel
+
+
+def test_JobsModel(app):
```
Comment (dtsehl) [inline tests/test_jobs.py:6]: Great job with this unit test.

**Review by dtsehl — APPROVED** [review]

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  api_key:
+    secure: 1/of71WopgZixVtKEupFs5ZqdvhcJ9caYTWG23TprGtuMBYmvlez5NuSOQnD1bCRrQntFixPtp5G0m9+R8thZ79t4J5a9X4bUdO3Zqt5cy3J8eOUI8zx0qs8AwhJva7LuPIYgQ9Ec4r3c6r54qJ/MvPABS3nJTiT6FDQMc13nzJ8uhzfZjLz53hXADcVC8LJGygO8wV0Wmem4/GOpHeo+u+PRnOFcfyJf7e7B+tLn5rxi6hDuA/T7fvkO9XSjHVcA6hUVAA0+5uUnMVb2JrjlxDkV9/iLuWWM2fVp7iiwjTfzLeectDP+br1htjvzcl76j4pAZITVqX8oHm5CcDbxMBafyqYhpeT4Ub28WsbfLiIuaLKN8EXfX3A88l+tZE8MuchS6tDgkS68C9G82NbKMrn1KgskITRx1Fx8p/k4eI6esSZo/iq4exNFJANj1jGa9fYixwN6hOfX/rCge9XjukAHWrwtan4NWN/hxy48Ger4zrMgTjIYrgQp3Sg6aK2NrxJlG7dH2YCHL5w1B3IMw86SjCmsfUvoWN0ySuflC/FWdmVgjS62MaTn/LChNmT7Y3iQjeP402++b08lgc5OZPTCCyUq0Gar13vINsaJxN/zO0CwSm6plSyOxPgJxQe4477v/tBSaT2MQYQWLsy8e8Wqkc7FMb72ofzP2/SXFA=
+  app: careerdaybe
+  on:
+    repo: Career-Day/career_day_BE
+    branch: main
+  skip_cleanup: 'true'
+  run:
+    - pip install -r requirements.txt
+    - python app.py db upgrade
```
Comment (janegreene) [inline .travis.yml:26]: We are creating and updating a postgres database for the CI environment.

**Review by janegreene — APPROVED** [review]
Excellent work!

**Pull request merged**